src/community/script/core/src/main/java/org/geoserver/script/rest/converter/ScriptConverter.java

```
@@ -0,0 +1,179 @@
+            if (checkPath(request)) {
+                try {
+                    link = URLDecoder.decode(link, "UTF-8");
+                    encodeAlternateAtomLinkNoExt(link, writer);
+                    writer.addAttribute("type", getMediaType());
+
+                    writer.endNode();
+                } catch (UnsupportedEncodingException e) {
+
+                }
+            } else {
+                encodeAlternateAtomLink(link, writer);
+            }
+        }
+
+        @Override
+        public void encodeCollectionLink(String link, HierarchicalStreamWriter writer) {
+            if (checkPath(request)) {
+                try {
+                    link = URLDecoder.decode(link, "UTF-8");
+                    encodeAlternateAtomLinkNoExt(link, writer);
+                    writer.addAttribute("type", getMediaType());
+
+                    writer.endNode();
+                } catch (UnsupportedEncodingException e) {
+
+                }
+            } else {
+                encodeAlternateAtomLink(link, writer);
+            }
+        }
+
+        @Override
+        public String getMediaType() {
+            return MediaType.APPLICATION_ATOM_XML_VALUE;
+        }
+
+        @Override
+        public String getExtension() {
+            return "xml";
+        }
+    }
+
+    /**
+     * JSON handling for Script lists
+     */
+    @Component
+    public static class JSONXStreamScriptListConverter extends JSONXStreamListConverter {
+        @Autowired
+        HttpServletRequest request;
+
+        @Override
+        public void encodeLink(String link, HierarchicalStreamWriter writer) {
+            if (checkPath(request)) {
+                try {
+                    link = URLDecoder.decode(link, "UTF-8");
+                    writer.startNode("href");
+                    writer.setValue(hrefNoExt(link));
+                    writer.endNode();
+                } catch (UnsupportedEncodingException e) {
+
```

You should at least log a WARNING or DEBUG message here, swallowing exceptions makes debugging unnecessarily difficult